src/paths.test.ts

```diff
@@ -16,24 +19,35 @@ describe("parentPath", () => {
   const paths: Array<[string, string | null]> = [
     ["app.hns/path/file.json", "app.hns/path"],
     ["app.hns///path///file.json", "app.hns/path"],
+    ["APP.hns///Path///file.json", "app.hns/Path"],
     ["app.hns//path", "app.hns"],
     ["app.hns/path/", "app.hns"],
+    ["/app.hns/path/", "/app.hns"],
```

Comment:
this should be null too, I think ?

Reply:
What "dataDomain" is this in? Is this a relative path, a null dataDomain or do we just ignore the leading '/'?
if 1, I don't know how to get an absolute path
2, How does an app get permissions to a null dataDomain?
3, I think it's fine to accept someone does this as a strange convention, but would it be better to just throw an error in development so they conform to path-style we expect?

Reply:
I am thinking we disallow the leading / for now. We can always allow it later if there's a use case. Allowing it now and then removing it later would be harder.